website/static/js/fangorn.js

```diff
@@ -754,6 +754,10 @@ function _fangornDropzoneSuccess(treebeard, file, response) {
  */
 var DEFAULT_ERROR_MESSAGE = 'Could not upload file. The file may be invalid ' +
     'or the file folder has been deleted.';
+// Which DEFAULT_ERROR_MESSAGE is best?
+// var DEFAULT_ERROR_MESSAGE = 'Unable to reach the provider, please try ' +
```

Comment:
Since this was the message that we were using before, let's go with this one as the default. You can remove the other one above.